cmd/system-probe/config/adjust.go

```diff
@@ -40,7 +41,7 @@ func Adjust(cfg model.Config) {
 		cfg.Set(netNS("enabled"), true, model.SourceAgentRuntime)
 	}
 
-	validateString(cfg, spNS("sysprobe_socket"), defaultSystemProbeAddress, ValidateSocketAddress)
+	validateString(cfg, spNS("sysprobe_socket"), setup.DefaultSystemProbeAddress, ValidateSocketAddress)
```

Comment:
Wondering if the default values should be part of the `defaultpaths` package 
cc @hush-hush